buildchain/buildchain/packaging.py

```diff
@@ -73,6 +73,12 @@ def task__package_mkdir_rpm_root() -> types.TaskDict:
         directory=constants.PKG_RPM_ROOT, task_dep=['_package_mkdir_root']
     ).task
 
+def task__package_mkdir_deb_root() -> types.TaskDict:
+    """Create the Debian packages root directory."""
+    return targets.Mkdir(
+        directory=constants.PKG_DEB_ROOT, task_dep=['_package_mkdir_root']
+    ).task
+
 def task__package_mkdir_iso_root() -> types.TaskDict:
```

Comment:
Something that we missed in the previous PR: this function should be renamed `task__package_mkdir_rpm_iso_root`

Reply:
I don't understand, the task `task__package_mkdir_rpm_iso_root` exist 6 lines after. 
The tasks tagged with an OS familly like `rpm` or `deb` call the tasks untagged like, in this case, `task__package_mkdir_iso_root`
Please tell me if I missed something

Reply:
Meh, go cheated by the Github display:

![meh](https://user-images.githubusercontent.com/34654278/63430563-28da4680-c41d-11e9-8887-db05d3a83934.png)

Didn't see that there was a line gap…
Everything is fine, forget it ^^